src/dev-mock-service/reddit-api-mock-service/index.ts

```diff
@@ -0,0 +1,506 @@
+  };
+  public getWikiPages: RedditAPIClient["getWikiPages"] = (...args) => {
+    return this.#defaultHandler("getWikiPages")(...args);
+  };
+  public createWikiPage: RedditAPIClient["createWikiPage"] = (...args) => {
+    return this.#defaultHandler("createWikiPage")(...args);
+  };
+  public updateWikiPage: RedditAPIClient["updateWikiPage"] = (...args) => {
+    return this.#defaultHandler("updateWikiPage")(...args);
+  };
+  public getWikiPageRevisions: RedditAPIClient["getWikiPageRevisions"] = (
+    ...args
+  ) => {
+    return this.#defaultHandler("getWikiPageRevisions")(...args);
+  };
+  public revertWikiPage: RedditAPIClient["revertWikiPage"] = (...args) => {
+    return this.#defaultHandler("revertWikiPage")(...args);
+  };
+  public getWikiPageSettings: RedditAPIClient["getWikiPageSettings"] = (
+    ...args
+  ) => {
+    return this.#defaultHandler("getWikiPageSettings")(...args);
+  };
+  public updateWikiPageSettings: RedditAPIClient["updateWikiPageSettings"] = (
+    ...args
+  ) => {
+    return this.#defaultHandler("updateWikiPageSettings")(...args);
+  };
+  public addEditorToWikiPage: RedditAPIClient["addEditorToWikiPage"] = (
+    ...args
+  ) => {
+    return this.#defaultHandler("addEditorToWikiPage")(...args);
+  };
+  public removeEditorFromWikiPage: RedditAPIClient["removeEditorFromWikiPage"] =
+    (...args) => {
+      return this.#defaultHandler("removeEditorFromWikiPage")(...args);
+    };
+  public getMessages: RedditAPIClient["getMessages"] = (...args) => {
+    return this.#defaultHandler("getMessages")(...args);
+  };
+  public markAllMessagesAsRead: RedditAPIClient["markAllMessagesAsRead"] = (
+    ...args
+  ) => {
+    return this.#defaultHandler("markAllMessagesAsRead")(...args);
+  };
+  public report: RedditAPIClient["report"] = (...args) => {
+    return this.#defaultHandler("report")(...args);
+  };
+  public getModQueue: RedditAPIClient["getModQueue"] = (...args) => {
+    // @ts-ignore
+    return this.#defaultHandler("getModQueue")(...args);
+  };
+  public getReports: RedditAPIClient["getReports"] = (...args) => {
+    // @ts-ignore
+    return this.#defaultHandler("getReports")(...args);
+  };
+  public getSpam: RedditAPIClient["getSpam"] = (...args) => {
+    // @ts-ignore
+    return this.#defaultHandler("getSpam")(...args);
+  };
+  public getUnmoderated: RedditAPIClient["getUnmoderated"] = (...args) => {
+    // @ts-ignore
+    return this.#defaultHandler("getUnmoderated")(...args);
+  };
+  public getEdited: RedditAPIClient["getEdited"] = (...args) => {
+    // @ts-ignore
+    return this.#defaultHandler("getEdited")(...args);
+  };
+  public getVaultByAddress: RedditAPIClient["getVaultByAddress"] = (
+    ...args
+  ) => {
+    return this.#defaultHandler("getVaultByAddress")(...args);
+  };
+  public getVaultByUserId: RedditAPIClient["getVaultByUserId"] = (...args) => {
+    return this.#defaultHandler("getVaultByUserId")(...args);
+  };
+}
+
+export const createDevvRedditApi = (
```

Comment:
it's a shame we need this helper but I'm not surprised. what's in RedditAPIClient that we have to cast to?